scripts/build.signer.sh

```diff
@@ -52,7 +55,8 @@ ic-wasm \
 
 # adds the content of $canister.did to the `icp:public candid:service` custom section of the public metadata in the wasm
 ic-wasm "$BUILD_DIR/signer.optimized.wasm" -o "$BUILD_DIR/signer.candid.wasm" metadata candid:service -f "$CANDID_FILE" -v public
-ic-wasm "$BUILD_DIR/signer.candid.wasm" -o "$BUILD_DIR/signer.metadata.wasm" metadata git_commit_id -f "$COMMIT_FILE" -v public
+ic-wasm "$BUILD_DIR/signer.candid.wasm" -o "$BUILD_DIR/signer.commit.wasm" metadata git_commit_id -f "$COMMIT_FILE" -v public
```

Comment:
Just sharing an idea here: Given `candid:service`, in Juno I also prefixed my metadata with `juno:build` - i.e. "something:otherthing" - so I was wondering if you would also prefix those? like `git:commit_id` and `git:tags` or something.

Reply:
Done.  Metadata is now:
```
icp:public candid:service
icp:public candid:args
icp:public git:commit <- was git_commit_id
icp:public git:tags <- was git_tags
```
The metadata is also declared in dfx.json, out of politeness although dfx doesn't add the metadata.